eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
@andrerom: in this case `$binaryFileId` is an absolute path in a filesystem (to an image file) and the goal of this test is to actually throw a `InvalidBinaryFileIdException`. Test that fails is [ImageIntegrationTest::testCreateContentUsingIdPropertyThrowsWarning](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php#L285) due to [this data](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php#L307)